cmd/mistryd/server.go

```diff
@@ -134,6 +144,12 @@ func (s *Server) HandleNewJob(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) handleNewJobAsync(j *Job, w http.ResponseWriter) {
+	s.Log.Printf("Scheduling %s...", j)
+	defer s.Work(context.Background(), j)
```

Comment:
We don't want to leave the connection open. You could just `go s.Work` this.

Reply:
yes, I meant to go this